MozillaFoundation/foundation.mozilla.org
# Homepage hero updates

# Description

This PR updates the homepage hero styles and features to match the redesigned specification.

## Current state:
![image](https://github.com/user-attachments/assets/9e57f8b7-53ed-49b8-b7d3-8ebfaa4d4912)
*Homepage*

![image](https://github.com/user-attachments/assets/dd8a8a6e-f961-4b20-a411-700bb1a9aed6)
*CMS*

## Updated state
![image](https://github.com/user-attachments/assets/d6b4596f-78a1-46e1-a3bd-0c5a554841c2)
*Homepage with updated layout*

![image](https://github.com/user-attachments/assets/f9eac6a8-06ca-4c51-9bc4-7bd69ab8786f)
*CMS with `Display hero button` checkbox*

# To test

1. Open the [preview app homepage](https://foundation-s-tp1-975-ad-ktitkx.herokuapp.com/en/).
2. Review homepage hero section updated layout for the title (bottom margin, column width)
3. Log in to the [preview app cms](https://foundation-s-tp1-975-ad-ktitkx.herokuapp.com/cms/) with `admin2:admin2` credentials.
4. [Open the homepage for editing](https://foundation-s-tp1-975-ad-ktitkx.herokuapp.com/cms/pages/46/edit/) and input a long hero headline text.
5. Disable the `Display hero button` checkbox. Publish
6. Review the [preview app homepage](https://foundation-s-tp1-975-ad-ktitkx.herokuapp.com/en/) to confirm the hero button being not displayed and the longer headline being updated.

Link to sample test page: https://foundation-s-tp1-975-ad-ktitkx.herokuapp.com/
Related PRs/issues: #12601

┆Issue is synchronized with this [Jira Story](https://mozilla-hub.atlassian.net/browse/TP1-1019)
